tests/acceptance/features/help-screen.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Help screen
+
+  As a tester
+  I want to know whether the HTML formatter is configured
+  So that I can verify I have integrated the formatter correctly.
+
+  Scenario:
```

Comment:
BETTER: Scenario: Ensure that HTML formatter is shown in list of formatters
Or similar ...